### DISCUSSION TITLE
power consumption/battery operation/life?

### DISCUSSION THREAD

**Discussion by eigenphase** [jomjol/AI-on-the-edge-device#1265] [General]
My water meter is outside on the public walkway under a cover.

I am thinking of using ESP32-CAM and attaching it to the cover. Issue is I can't really run a cable.

Has anyone tried powering this via a battery? Which solution did you use and how long was the battery life?

**Comment by caco3** [comment]
The device is not power optimized, I don't think you get far with running form battery.

Please also have a look on:
 - https://github.com/jomjol/AI-on-the-edge-device/issues/188
 - https://github.com/jomjol/AI-on-the-edge-device/issues/234

**Comment by jomjol** [comment]
Based on this: https://time4ee.com/articles.php?article_id=126 I assume, that we cannot go below 1.5mA without hardware modification.

Assuming a 2000mAh we can run up to 1300h or 55 days in idle mode. Knowing, that the caluclation take several minutes and is very power consuming it will be less, depending how often you want to read out.

For a monitoring device I don't think, that it is accepted, to replace the battery every 1-2 month, but expected to go for at least a year or even longer. Therefore I decided not to follow this approach up to now.
